spec/helpers.rb

```diff
@@ -2,4 +2,12 @@ module Helpers
   def json
     JSON.parse(response.body).with_indifferent_access
   end
+
+  def auth_headers
+    @auth_headers ||= user.create_new_auth_token
+  end
+
+  def create_auth_headers
```

Comment:
Given ` #auth_headers` I think ` #create_auth_headers` is not necessary

Reply:
Actually, it is! Maybe I named it wrong, it should be something like `#refresh_auth_headers` since in the case of logout the headers created previously are no longer valid for authentication